src/language/code_generator.py

```diff
@@ -194,6 +192,23 @@ def workload(self):
                         )
                         tasks.append(task)
                         yield task
+                elif filepath == pynode_cpp_tpl:
+                    chunk_length = math.ceil(len(self.nodes) / num_pybind_files)
```

Comment:
Can you not just use the `//` operator (`+ 1`)?

Reply:
As a direct substitution, that gives a different answer if `num_pybind_files` divides `len(self.nodes)` exactly? To me it's also less expressive, but I can change it if you insist :)

Reply:
nope, I don't insist. It's fine.